src/scenario.jl

```diff
@@ -668,12 +670,18 @@ function run_model(domain::Domain, param_set::YAXArray, functional_groups::Vecto
             site_data.area .* site_data.k, (1, 1, n_locs)
         )
 
+        # Constrain growth by available area
+        # Constraint Coefficient -> log(1 + left over space)
```

Comment:
Is this an unnecessary (and outdated) comment?

Reply:
I was concerned the code didn't make the formula obvious, but it can be removed or updated to reflect the base of the log.

Reply:
Why not use `log2()` ? Wouldn't that make the code clearer?